systemtest/src/test/java/io/strimzi/systemtest/CruiseControlST.java

```diff
@@ -35,20 +49,111 @@ void setup() {
         applyRoleBindings(NAMESPACE);
         // 050-Deployment
         KubernetesResource.clusterOperator(NAMESPACE).done();
+
+        KafkaResource.kafkaWithCruiseControl(CLUSTER_NAME, 3, 3).done();
     }
 
     @Test
-    void testCCDeployment()  {
-        KafkaResource.kafkaWithCruiseControl(CLUSTER_NAME, 3, 3).done();
+    void testCruiseControlDeployment()  {
+        String ccStatusCommand = "curl -X GET localhost:" + CRUISE_CONTROL_DEFAULT_PORT + CRUISE_CONTROL_ENDPOINT;
 
-        String ccStatusCommand = "curl -X GET localhost:9090/kafkacruisecontrol/state";
         String ccPodName = PodUtils.getFirstPodNameContaining("cruise-control");
-        LOGGER.info("Using pod: " + ccPodName);
+
         String result = cmdKubeClient().execInPodContainer(ccPodName, "cruise-control",
-                "/bin/bash", "-c", ccStatusCommand).out();
+            "/bin/bash", "-c", ccStatusCommand).out();
+
+        LOGGER.info("Verifying that {} is running inside the Pod {} and REST API is available", CRUISE_CONTROL_NAME, ccPodName);
 
         assertThat(result, not(containsString("404")));
         assertThat(result, containsString("RUNNING"));
+
+        verifyThatCruiseControlTopicsArePresent();
+    }
+
+    @Test
+    void testInstallationAndCreatedTopics() {
+
+        String clusterName = "second" + CLUSTER_NAME;
+
+        KafkaResource.kafkaEphemeral(clusterName + "1", 3, 1).done();
+
+        KafkaResource.replaceKafkaResource(clusterName, kafka -> {
```

Comment:
Why do you replace kafka instead of put CC directly into first deploy?

Reply:
Already discussed, I have reworked it. Now i using the shared kafka where i firstly remove the CC and the add him...